<!-- -->
<launch>

  <arg name="config_path"    default="$(find perception)"/>
  <!--      launch-prefix="xterm -e gdb -ex run args " -->>
  <node pkg="perception" type="perception" name="perception_01" output="screen">
    <param name="config_path"     value="$(find perception)" />
  </node>

  <node pkg="detected_objects_visualizer" 
      type="visualize_detected_objects" 
      name="cnnseg_detect_visualization_02"
      output="screen" ns="/detection/lidar_detector" />

  <node pkg="detected_objects_visualizer" 
      type="visualize_detected_objects" 
      name="track_objects_visualization_02"
      output="screen" ns="/detection/object_tracker" />

</launch>
